src/agent/inspectordebugapi.ts

```diff
@@ -46,9 +46,12 @@ export class InspectorDebugApi implements debugapi.DebugApi {
   breakpoints: {[id: string]: BreakpointData} = {};
   sourcemapper: SourceMapper;
   session: inspector.Session;
-  scriptmapper: {[id: string]: any} = {};
-
   listeners: {[id: string]: utils.Listener} = {};
+  scriptmapper: {[id: string]: any} = {};
```

Comment:
For `scriptmapper`, `locationmapper`, and `breakpointmapper`, could a `Map<>` be used instead of an Object.

Reply:
Map.prototype.get returns value or undefined. This makes whereever we want to use the code for get be casting to the actual value type, and makes it complex. I added a TODO here and once we have a wrapper of Map class to implement Map.prototype.get returns the actually value (or throw exception), and we can replace all map object in the code to be Map.